libguard/include/guard_record.hpp

```diff
@@ -84,6 +84,8 @@ enum GardType
     GARD_Power = 0xE9,
     GARD_PHYP = 0xEA,
     GARD_Reconfig = 0xEB,
+    GARD_Sticky_deconfig = 0xEC, // Force deconfig on reconfig loop,
+                                 // not availible for resource recovery.
```

Comment:
is this correct?

Reply:
Yeah, referred to the host code on this description since the host app decides the usage of the type guard record based on the use case.

Reply:
good